discord_thread_summarizer_using_openai/main.bal

```diff
@@ -0,0 +1,73 @@
+import ballerina/io;
+import ballerina/log;
+import ballerina/time;
+
+const TYPE_FORUM_CHANEL = 15;
+const time:Seconds ONE_DAY_IN_SECONDS = 86400;
+configurable int forumChannelId = ?;
+
+function readThreads(ActiveThreads activeThreads) returns error? {
+
+    foreach ChannelThread thread in activeThreads.threads {
+
+        // check if the thread has any recent messages
```

Comment:
Also doesn't add value IMO. Self-explanatory? Please check all comments.

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/3ae658014f3ffe67497928faf8e7e99fc01b6c54